applications/luci-app-babeld/htdocs/luci-static/resources/babeld.js

```diff
@@ -0,0 +1,171 @@
+function renderTableXRoutes(ubus_data, target_id) {
+	var data = JSON.parse(ubus_data);
```

Comment:
... only to parse it back to JSON here? You should directly pass the JSON.

Reply:
Actually when I was passing the JSON directly it was causing [object Object] error like this->renderTableInfo([object Object], 'babeld')

Reply:
Ahh, yeah because you're not actually calling `renderTableInfo()` but you generate code that is calling the function later. Via indirect eval... Use the same approach for `babeld.js` as for `olsr-viz.js` suggested in the other PR, that is do a promise-wrapped loading of `babeld.js` in the `load()` function. Then you can significantly simplify all subsequent code as you can call functions such as `renderTableInfo()` directly.

Reply:
Okay! Sure

Reply:
I fixed it & made the code shorter. Thanks